### PR TITLE
10 implement community page designs

### LINKED ISSUE #10: Implement community page designs

### LINKED ISSUE #12: Set tabindex on modal to focus on signup form

### PR DESCRIPTION
**Pull request by rickymetz**
Closes #10
Closes #12

### CODE REVIEW

**Comment by rickymetz** [comment]
Animation is not in this PR, but is captured in a separate issue ( #14 ) in order to not delay production.

File: nycedu-react.sublime-workspace [hunk 1]
```diff
@@ -0,0 +1,683 @@
+{
```
Comment (seanperkins) [inline nycedu-react.sublime-workspace]: Remove this from the project.

File: src/community/Community.js [hunk 2]
```diff
@@ -9,9 +15,63 @@ class Community extends Component {
     return (
       <div className="community-page container">
         <DocumentTitle title="Our community | #NYCEDU" />
+        <object id="sky-background" className="community-backgrounds" type="image/svg+xml" data={sky}>Your browser does not support svgs.</object>
         <h1>Community</h1>
         <p>If you care about education in NYC, then you are already a member of #NYCEDU! Meet some of your fellow community members.</p>
-        <SignupForm handleSubmission={this.props.handleToast} />
+        <div className="member-container">
```
Comment (seanperkins) [inline src/community/Community.js]: Create a data structure to hold the community members, then .map over it to generate the markup.

**Review by seanperkins — CHANGES_REQUESTED** [review]
Please run those images through something to optimize them.

File: src/community/Community.js (hunk 2 above)
Comment (rickymetz) [reply src/community/Community.js]: Fixed in cf57398

File: src/community/Community.js [hunk 3]
```diff
@@ -1,20 +1,72 @@
...
+      {
+        name: "Eugene Leventhal",
+        imageUrl: eugene,
+        buildCommunity: "I started the Bronx Education Meetup group to help connect those involved in education in the Bronx. I’m also working on EduDAO, a blockchain educational funding project.",
+        getInvolved: <p>Sign up for the <a href="https://www.meetup.com/Bronx-Education-Meetup/">meetup</a> or email me at <a href="mailto:eugene.leventhal@gmail.com">eugene.leventhal@gmail.com</a>.</p>
+      },
+      {
+        name: "Sean Perkins",
+        imageUrl: sean,
+        buildCommunity: "I started the Bronx Education Meetup group to help connect those involved in education in the Bronx. I’m also working on EduDAO, a blockchain educational funding project.",
```
Comment (seanperkins) [inline src/community/Community.js]: Build Community: I am working to ensure the technology we create for NYCEDU stays open and maintainable by the community. I also like to help folks that are new to software development get started in the field.

File: src/community/Community.js [hunk 4]
```diff
@@ -1,20 +1,72 @@
...
+        getInvolved: <p>Sign up for the meetup or email me at eugene.leventhal@gmail.com.</p>
```
Comment (seanperkins) [inline src/community/Community.js]:
```
<p>Check out the #NYCEDU <a target="_blank" href="https://github.com/NYCEDU/">Github repo</a> to see what's being created by our community. Interact with others in the <a target="_blank" href="https://nycedu.slack.com/messages/we_are_developers/">#we_are_developers</a> channel in Slack.
```

**Review by seanperkins — APPROVED** [review]

**Pull request merged**